templates/repo/settings/options.tmpl

```diff
@@ -201,8 +201,8 @@
 								</td>
 							</tr>
 						</tbody>
+						{{end}}{{/* end if: IsMirror */}}
 					</table>
-					{{end}}{{/* end if: IsMirror */}}
```

Comment:
Are you sure, the start `{{if .Repository.IsMirror}}` block is before `<table>`, so I think this change will leave a broken `</table>` end tag.

Reply:
Hmm, let me double check

Reply:
Well, the real "end" is here .... the template is totally broken. Let me try to fix it again.

![image](https://github.com/go-gitea/gitea/assets/2114189/609b621e-3ebe-4979-a105-3fdccff37fba)

Reply:
Ok, also update the comment to say `end if .Repository.IsMirror`

Reply:
Done in 76bcdfe527b8ce7b4770931667cc6974bf7bbe3b